src/Squirrel/UpdateManager.CheckForUpdates.cs

```diff
@@ -62,33 +62,28 @@ public CheckForUpdateImpl(string rootAppDirectory)
                     }
 
                     this.Log().Info("Downloading RELEASES file from {0}", updateUrlOrPath);
-
-                    int retries = 3;
-
-                retry:
-
-                    try {
-                        var uri = Utility.AppendPathToUri(new Uri(updateUrlOrPath), "RELEASES");
-
-                        // Not working with custom queries
-                        /*if (latestLocalRelease != null) {
-                            uri = Utility.AddQueryParamsToUri(uri, new Dictionary<string, string> {
-                                { "id", latestLocalRelease.PackageName },
-                                { "localVersion", latestLocalRelease.Version.ToString() },
-                                { "arch", Environment.Is64BitOperatingSystem ? "amd64" : "x86" }
-                            });
-                        }*/
-
-                        var data = await urlDownloader.DownloadUrl(uri.ToString());
-                        releaseFile = Encoding.UTF8.GetString(data);
-                    } catch (WebException ex) {
-                        this.Log().InfoException("Download resulted in WebException (returning blank release list)", ex);
-
-                        if (retries <= 0) throw;
-                        retries--;
-                        goto retry;
-                    }
-
+                    
+                    var uri = Utility.AppendPathToUri(new Uri(updateUrlOrPath), "RELEASES");
+
+					// Not working with custom queries
+					/*if (latestLocalRelease != null) {
+                        uri = Utility.AddQueryParamsToUri(uri, new Dictionary<string, string> {
+                            { "id", latestLocalRelease.PackageName },
+                            { "localVersion", latestLocalRelease.Version.ToString() },
+                            { "arch", Environment.Is64BitOperatingSystem ? "amd64" : "x86" }
+                        });
+                    }*/
+
+					string tempFilePath = Path.GetTempFileName();
```

Comment:
I'd just run autoformat on the whole file...

Reply:
Fixed.